components/rmrk/Create/SimpleMint.vue

```diff
@@ -580,7 +576,8 @@ export default class SimpleMint extends mixins(
     originalBlockNumber: string
   ): Promise<void> {
     try {
-      const { version, price } = this
+      // TODO: WORK WITH V2
```

Comment:
still need to do it?

Reply:
https://github.com/kodadot/nft-gallery/issues/5645